oleggerasimovich/src/parser.py

```diff
@@ -0,0 +1,37 @@
+import argparse
+import json
+import os
+from argparse import Namespace
+
+from dotenv import load_dotenv
+from pymongo import MongoClient
+
+def cli() -> Namespace:
+    parser = argparse.ArgumentParser()
```

Comment:
Also, it is better to leave some description of your script in CLI. You can make this with the `description` parameter of `ArgumentParser`.